clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -178,4 +192,55 @@ public static Properties producerConfig(final String bootstrapServers,
         properties.putAll(additional);
         return properties;
     }
+
+    public static Properties producerConfig(final String bootstrapServers, Class keySerializer, Class valueSerializer) {
+        return producerConfig(bootstrapServers, keySerializer, valueSerializer, new Properties());
+    }
+
+    public static Properties consumerConfig(final String bootstrapServers,
+                                            final String groupId,
+                                            final Class keyDeserializer,
+                                            final Class valueDeserializer,
+                                            final Properties additional) {
+
+        final Properties consumerConfig = new Properties();
+        consumerConfig.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
+        consumerConfig.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
+        consumerConfig.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        consumerConfig.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, keyDeserializer);
+        consumerConfig.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, valueDeserializer);
+        consumerConfig.putAll(additional);
+        return consumerConfig;
+    }
+
+    /**
+     * returns consumer config with random UUID for the Group ID
+     */
+    public static Properties consumerConfig(final String bootstrapServers, Class keyDeserializer, Class valueDeserializer) {
+        return consumerConfig(bootstrapServers,
+                              UUID.randomUUID().toString(),
+                              keyDeserializer,
+                              valueDeserializer,
+                              new Properties());
+    }
+
+    /**
+     *  uses default value of 30 seconds for timeout
+     */
+    public static void waitForCondition(TestCondition testCondition) throws InterruptedException {
+        waitForCondition(testCondition, 30000);
```

Comment:
Is this a good default timeout? The default timeout in `TestUtils.waitUntilTrue` is 5 seconds.